src/partials/header-scripts.hbs

```diff
@@ -1,6 +1,3 @@
 {{#unless page.attributes.disabletracking}}
```

Comment:
You can even remove the condition `{{#unless page.attributes.disabletracking}}`.
The scripts that we might add in the future will not necessarily be tracking scripts.

Reply:
Done